### PR TITLE
Create .editorconfig

### PR DESCRIPTION
**Pull request by steveszc**
basic .editorconfig for middleman

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+[*]
+indent_style = space
+indent_size = 2
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.md]
+trim_trailing_whitespace = false
```
Comment (michaelrkn) [inline .editorconfig]: why false here?

Comment (steveszc) [inline .editorconfig]: I just lifted the [.editorconfig from the middleman project](https://github.com/middleman/middleman/blob/master/.editorconfig)... I don't have a strong preference either way.

Comment (gmurphey) [inline .editorconfig]: Spaces are sometimes significant in markdown (double-space at end of a line creates a newline).

**Comment by michaelrkn** [comment]
Thanks!!

**Pull request merged**